### ISSUE TITLE
doesn't pick up transpiler: 'babel' in system.conf.js

### ISSUE THREAD

**Issue by edrex** [rolaveric/karma-systemjs#21]
Settings in `system.conf.js` should have parity with the `systemjs.config` in `karma.conf.js`, but `transpiler: 'babel'` in `system.conf.js` isn't picked up by karma-systemjs:

```
Cannot find "traceur".
  Did you forget to install it ?
  npm install traceur --save-dev
```

(The setting has to be in `karma.conf.js` for the detection to work)

**Comment by rolaveric** [comment]
Just tested against my [angular-seed fork](https://github.com/rolaveric/angular-seed/tree/es6) and it's definitely using the transpiler specified in the SystemJS config.
Did you specify the config file under `systemjs.configFile` in your karma config?

**Comment by edrex** [comment]
I whittled down my project a bit to make [a test case](https://github.com/edrex/test-karma-systemjs-babel), where `karma start` is failing after the warning above. Adding `transpiler: 'babel'` to `systemjs.config` allows it to run without an error.

**Comment by rolaveric** [comment]
Ah, I think I see the problem.  It's the multiple System.config() calls that jspm creates.  I thought that had been fixed, but maybe it just masked the problem.
I'll fix it up now.

**Comment by rolaveric** [comment]
Just committed b118c7c117460adce48297bfebf34d38ddd9b53c and published v0.3.1 which should fix it.
